pkg/engine/context/informerCache.go

```diff
@@ -0,0 +1,63 @@
+package context
```

Comment:
This package should be named `resolvers`

Reply:
fixed